src/xdocs/config_imports.xml

```diff
@@ -626,6 +639,35 @@ import android.*;
     &lt;property name=&quot;illegalPkgs&quot; value=&quot;java.io, java.sql&quot;/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check so that it rejects classes <code>java.util.Date</code> and <code>java.sql.Connection</code>:
+        </p>
+        <source>
+&lt;module name=&quot;IllegalImport&quot;&gt;
+    &lt;property name=&quot;illegalClasses&quot; value=&quot;java.util.Date, java.sql.Connection&quot;/&gt;
+&lt;/module&gt;
+        </source>
+
+        <p>
+          To configure the check so that it rejects packages not satisfying to regular expression <code>^java\.util.*</code>:
+        </p>
+        <source>
+&lt;module name=&quot;IllegalImport&quot;&gt;
+    &lt;property name=&quot;regexp&quot; value=&quot;true&quot;/&gt;
+    &lt;property name=&quot;illegalPkgs&quot; value=&quot;^java\.util.*&quot;/&gt;
+&lt;/module&gt;
+        </source>
```

Comment:
please add code chunk example to show user by trailing comment "//violation" where violation will be placed.
Make examples with valid and invalid code.